stress/shell/src/main/java/alluxio/stress/cli/fuse/FuseIOBench.java

```diff
@@ -68,11 +70,16 @@ public static void main(String[] args) {
 
   @Override
   public void prepare() throws Exception {
+    if (mParameters.mNumFiles % mParameters.mNumDirs != 0) {
+      throw new IllegalArgumentException(String
+          .format("Number of files (%d) must be a multiple of the number of directories (%d)",
```

Comment:
I don't like the idea of forcing users to put two parameters that must be a multiplier. Every user will not be happy seeing that. 
What i recommend before is 
- we ask users to provide numDirs, numFilesPerDir, numDirs is better to be a multiplier of the thread number, but it's not mandary. 

Yes, we can simplify our developing logic by making users a little harder to use, but making them super hard to use should be avoided.

Reply:
I see. Yeah this sounds more reasonable. Thanks!